src/shared/utilities/vsCodeUtils.ts

```diff
@@ -6,100 +6,118 @@
 'use strict'
 import * as vscode from 'vscode'
 import * as nls from 'vscode-nls'
-export const localize = nls.loadMessageBundle()
 
-import { ErrorOrString, getLogger, Logger, LogLevel } from '../logger'
+export const localize: TemplateParser = nls.loadMessageBundle()
 
-// ------- Experimental combined output channel & traditional logger  --------------
+// TODO: initialize NLS here rather than extension.ts. Prefer packages to import localize from here
+// Advantages:
+//  • Ensure we only call nls.config once
+//  • Don't have to call nls.loadMessageBundle() in every file
 
-interface LogParams {
-    args: ErrorOrString[],
-    channel: vscode.OutputChannel,
-    level: LogLevel,
-    logger: Logger,
-    msg: string,
+import { BasicLogger, ErrorOrString, getLogger, LogLevel } from '../logger'
```

Comment:
Is it possible to group all of the import statements together?

Reply:
✔️